bamboo/audit.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -ex
+. ./bamboo/use-working-directory.sh
```

Comment:
Do we have a ticket to refactor these scripts to something a bit more manageable? I am NOT suggesting any part of that needs to be done here. Just want to be sure we capture the intention.

Reply:
I don't believe so, no.      I'd expect that probably solidly falls into the sort of ticket we wouldn't take into the backlog, but I'm not opposed to working a ticket that gets closed for documentation reasons.

Reply:
Talked about this separately. I'm sorta in the "if it's not worth doing now, it's not worth doing" camp.